### PR TITLE
Direct Messages week 2

### PR DESCRIPTION
**Pull request by Oporto**
Implemented step 5 and 6 of direct messages familiarization project

### CODE REVIEW

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 1]
```diff
@@ -52,12 +52,12 @@ public void storeMessage(Message message) {
    * @return a list of messages posted by the user, or empty list if user has never posted a
```
Comment (pnvt23) [inline src/main/java/com/google/codeu/data/Datastore.java]: can you also update the documentation, since the behavior of the method changes

**Comment by pnvt23** [comment]
Can you check the merge conflict as well? Thanks

File: src/main/java/com/google/codeu/data/Datastore.java (hunk 1 above)
Comment (Oporto) [reply src/main/java/com/google/codeu/data/Datastore.java]: Updated

**Comment by Oporto** [comment]
Solved merged conflicts with commits from message feed. Updated its logic of getting all messages vs by a specific user to get messages by recipients and not users that posted them

**Review by pnvt23 — APPROVED** [review]

**Pull request merged**